### DISCUSSION TITLE
How to implement Celery with Supabase?

### DISCUSSION THREAD

**Discussion by lienista** [supabase/supabase#3315] [General]
Hi, 

How do you automate celery to perform such tasks such as, sending email and text notifications, with Supabase

**Comment by silentworks** [comment]
Can you clarify what celery is first please? and provide a link to it if possible.

**Reply by lienista** [reply]
Celery is a task automation library: https://docs.celeryproject.org/en/stable/getting-started/introduction.html

How do you automate sending emails and sms at specific times to your users? I'm also open to using RabbitMQ or any other library, as long as I can automate these tasks

**Reply by aristidesnakos** [reply]
@lienista did you ever implement a supabase client + celery solution ? I am thinking it has to be a websocket + celery implementation, which Supabase doesn't support : https://github.com/orgs/supabase/discussions/21204#discussioncomment-8448000

**Comment by lienista** [comment]
I did not implement celery on supabase. It’s just a toy and it was unstable at the time. I didn’t see the point of having supabase whereas you can use PostgreSQL directly. You can also use vercel/postgesql and everything is possible, won’t have to deal with scaling issues w PostgreSQL. On Feb 16, 2024, at 1:01 AM, Ari ***@***.***> wrote:﻿
@lienista did you ever implement a supabase client + celery solution ? I am thinking it has to be a websocket + celery implementation, which Supabase doesn't support : https://github.com/orgs/supabase/discussions/21204#discussioncomment-8448000

**Comment by aristidesnakos** [comment]
Thanks ! Didn't think to check with Vercel -- makes sense.
